CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ The format is based on the [KeepAChangeLog] project.
 
 [KeepAChangeLog]: http://keepachangelog.com/
 
+## 0.13.0 [Unreleased]
+
+### Fixed
+- [#427] Dealing with unordered response_types
+- [#399] Matching response_types for authz requests is too strict
```

Comment:
Better would be - 'Made matching for response_types less strict for authorization requests', or something more friendly for poor person who comes here looking for what changed :)

Reply:
OK done